docs/modules/reference/pages/glossary.adoc

```diff
@@ -167,19 +180,32 @@ Requisitions:: Sets of nodes to import into {page-component-title} for monitorin
 You can build requisitions iteratively and import them at a later date.
 (See xref:operation:provisioning/getting-started.adoc#requisition-create[Create a Requisition].)
 
-Round Robin Database (RRD):: Special type of database designed for collection, visualization, and analysis of time-series data.
+Resource:: Any hardware or software that {page-component-title} can access.
+
+Round Robin Database (RRD):: A special type of database designed for collection, visualization, and analysis of time-series data.
 Data is maintained as a fixed-size circular-buffer that overwrites the oldest data with new data.
 
+Sample:: A small set of data collected from a {page-component-title}-monitored source.
+
 Sentinel:: Karaf container that provides scalability for data processing of flow data.
 It also supports thresholding for streaming telemetry if you are using the xref:deployment:time-series-storage/newts/introduction.adoc[Newts] time-series strategy.
 (See xref:deployment:sentinel/introduction.adoc[Sentinel].)
 
+Service:: Software that performs a specified set of tasks (automation, listening for data requests, etc.).
+
+Simple Mail Transfer Protocol (SMTP):: An internet standard protocol.
+{page-component-title} monitors SMTP availability on network nodes to ensure that email messages can be sent and received.
+(See xref:reference:service-assurance/monitors/MailTransportMonitor.adoc[MailTransportMonitor].])
+
+Simple Network Management Protocol (SNMP):: An internet standard protocol used by {page-component-title} to collect, organize, and modify information about managed devices on an IP network.
+
 Snaps:: Self-contained software packages that run in a sandbox and have mediated access to host systems.
 Our Appliance Service uses snaps to distribute operating system packages and Minion service updates.
 
+Southbounder:: An alarm generated by {page-component-title} that is sent to a lower-level component (service, resource, etc.).
```

Comment:
First time I've heard this term used. 😀

Reply:
the existence of "northbound" implies the existence of "southbound"
https://github.com/OpenNMS/opennms/blob/7e9f5759badaf5636ae487a72b605a2a89e3e13e/opennms-alarms/api/src/main/java/org/opennms/netmgt/alarmd/api/support/AbstractNorthbounder.java
> AbstractNorthBounder
<p>The purpose of this class is manage the queue of alarms that need to be forward and receive queries to/from a Southbound Interface.</p>
<p>It passes Alarms on to the forwardAlarms method implemented by base classes in batches as they are added to the queue.  The forwardAlarms method does the actual work of sending them to the Southbound Interface.</p>
<p>Preserve, accept and discard are called to add the Alarms to the queue as appropriate.</p>

Reply:
Looks like we don't have anything called a Southbounder within the products, and it's not a term used often, so I think we can leave this one out for now. :)